hypervisor.go

```diff
@@ -272,3 +262,57 @@ func getHostMemorySizeKb(memInfoPath string) (uint64, error) {
 
 	return 0, fmt.Errorf("unable get MemTotal from %s", memInfoPath)
 }
+
+func runningOnVMM(cpuInfoPath string) (bool, error) {
```

Comment:
This is a useful function. How about making it public to allow consumers of virtcontainers to also make use of it?